assets/javascripts/climate-vulnerability-maps.js

```diff
@@ -1,13 +1,7 @@
-// const projection = d3.geoAlbers()
-//   .scale(120000)
-//   .rotate([71.057, 0])
-//   .center([-0.021, 42.38])
-//   .translate([960 / 2, 500 / 2]);
-
 function createJobMap(data) {
```

Comment:
I'm guessing this is from Perfect Fit---could you rename the function and the `parkingMap` variable  to more closely align with climate vulnerability (perhaps like `vulMap`?)